### ISSUE TITLE
Create a basic "logic" device

### ISSUE THREAD

**Issue by HansenChristoffer** [HansenChristoffer/DeviceConnectivityPlatform#20]
For now we can use a Raspberry PI just to have something that the backend or "The Bridge".

It should be able to receive a PING command and also acknowledge that PING right back to the bridge.

**Comment by HansenChristoffer** [comment]
This will 100% be the next step. We need some kind of basic hardware so that we can test the backend transceiver, unit tests and cheap hacks only works that far.
This issue is still a big one, as I will need to implement the security parts in C on the device.

**Comment by HansenChristoffer** [comment]
Will create separate issue for investigating this. Will need more details and a more structured map on how to proceed, before implementing any work. This issue will be put on hold and be blocked by the new issue, #32.
